js/popup.js

```diff
@@ -1,11 +1,49 @@
-let pupwin = document.createElement('section');
-pupwin.className='modlpup'
-pupwin.innerHTML = '<h3>Multi-post stories</h3>'
-+'<section class="prj1_tch_dtls">'
+const info = [{
+  title: 'Multi-post stories',
+  tch_dtls: ['css', 'html', 'bootstrap', 'javascript'],
+  img_src: 'images/popup_img_mob.png',
+  des_txt: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt quibusdam odit quis saepe fuga porro inventore ipsum cum. Nulla velit aspernatur nesciunt quo eum nobis. Non neque ut accusantium tempora?0',
+}];
+
+const backWin = document.createElement('section');
+backWin.className = 'bck_win';
+const pupWin = document.createElement('section');
+pupWin.className = 'modlpup';
+pupWin.style.overflow = 'scroll';
+pupWin.innerHTML = '<section class="pup_txt">'
+    + `<h3>${info[0].title}</h3>`
+        + '<ul>'
+            + `<li>${info[0].tch_dtls[0]}</li>`
+            + `<li>${info[0].tch_dtls[1]}</li>`
+            + `<li>${info[0].tch_dtls[2]}</li>`
+            + `<li>${info[0].tch_dtls[3]}</li>`
+        + '</ul>'
+        + `<img src=${info[0].img_src} alt="project display image">`
+    + '</section>'
+    + '<section class="pupdesc">'
+        + `<p>${info[0].des_txt}</p>`
+        + '<section class="pupbtns">'
+            + '<button class="see_live">See Live<img src="images/see_live_btn_icon.png" alt="see live project button"></button>'
+            + '<button class="see_source">See Source<img src="images/Icon -GitHub.png" alt="See source code on github button"></button>'
```

Comment:
- [ ] Please use the correct color for the model buttons and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A424) design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/196784911-db6e0f29-a586-44a2-b761-c4f4658d29d5.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/196784973-4a48574f-7752-4c90-b87c-1d2e142ed0d6.png)

Reply:
These were changed only for accessibility purposes and it was not accepted for accessibility when I used the same color scheme as in Figma.